spec/lib/rex/proto/http/client_spec.rb

```diff
@@ -742,4 +742,44 @@ def excuse_needs_auth
       expect(request.to_s).to eq(expected)
     end
   end
+
+  context 'with HttpTrace logging'
+    let(:sample_request) {
+      "GET / HTTP/1.1\nHost: www.google.com"
+    }
+    let(:sample_response) {
+      "HTTP/1.1 302 Found\nLocation: https://www.google.com/?gws_rd=ssl"
+    }
+    let(:http_trace_proc) { 
+      proc { |request, response|
+        puts("#" * 20)
+        puts("# Request : ")
```

Comment:
```suggestion
        puts("# Request:")
```

Applies for the rest, just looks a bit untidy to have trailing spaces and whitespace like this. Additionally this new format better matches the existing proc format for printing output that we have in our current HTTPTrace implementation.